main.go

```diff
@@ -312,10 +368,10 @@ func main() {
 		}
 
 		res, err := k.HandleCard(ev.UID)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, err)
+		if res != nil {
+			res.Print(lcd)
 		} else {
-			fmt.Println(res)
+			flashLCD(lcd, err.Error(), 255, 0, 0)
```

Comment:
Not sure. In general, I agree, but it might be, that all existing errors are usefull to pass to the user. I added a TODO, for now this is good enough, I think.